sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -268,12 +270,15 @@ def list_experiments(self, namespace=None):
         try:
             katibexp = thread.get(constants.APISERVER_TIMEOUT)
             result = []
-            for i in katibexp.get("items"):
-                output = {}
-                output["name"] = i.get("metadata", {}).get("name")
-                output["status"] = i.get("status", {}).get(
-                    "conditions", [])[-1].get("type")
-                result.append(output)
+            if in_short:
+                for i in katibexp.get("items"):
+                    output = {}
+                    output["name"] = i.get("metadata", {}).get("name")
+                    output["status"] = i.get("status", {}).get(
+                        "conditions", [])[-1].get("type")
+                    result.append(output)
+            else:
+                result = katibexp.get("items")
```

Comment:
Maybe we can cast the result to [`[]V1beta1Trial`](https://github.com/kubeflow/katib/blob/778e4fdf8d995a00f37ba06e4e5c3b768152f1ed/sdk/python/v1beta1/kubeflow/katib/models/v1beta1_trial.py#L21) list of objects ?
What do you think ?

Reply:
That would be better for safe usage!
However, then don't we have to change return types for other similar cases ?
i.e. `get_experiment`, `get_suggestion`, `list_experiments`

Reply:
For now, I changed list_experiments and list_trials to return List of v1beta1_objects rather than dict

Reply:
> However, then don't we have to change return types for other similar cases ?
i.e. get_experiment, get_suggestion, list_experiments

Yeah, that should be our final goal 😄 
We can change `get_experiment`, `get_suggestion`, etc. in the following PRs.

Reply:
That makes sense :)
Then I'll change return type to Union[List[dict], List[V1beta1Experiment]] until `in_short` parameter will be deprecated